### ISSUE TITLE
Nothing displayed in Listen to Events

### ISSUE THREAD

**Issue by vk2him** [xannor/ha_reolink_discovery#3]
Nothing is ever displayed when I listen to reolink_discovery however it did allow my Reolink RLC-822a camera to be setup

![Image 002](https://user-images.githubusercontent.com/22878629/185271802-9ea3c34b-7e39-49fe-822e-83ef4e2fbf71.jpg)
![Image 001](https://user-images.githubusercontent.com/22878629/185272100-2cdb2c0d-4b68-4632-a1f6-30aca867d592.jpg)

**Comment by xannor** [comment]
I had removed the event code because I found an alternate method that both avoids the serialization/de-serialization of events and would ensure an integration was loaded even if not configured. By default integrations are only loaded if they have a configuration.yaml entry or a config entry, or are loaded by the discovery system.

I could add an option to enable the firing of an event in addition, just hadn't thought of that at the time.
